fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/book/book1/Section10.java

```diff
@@ -0,0 +1,75 @@
+package ch.bbw.fabbwled.lands.book.book1;
+
+
+import ch.bbw.fabbwled.lands.book.SectionHandler;
+import ch.bbw.fabbwled.lands.book.SectionId;
+import ch.bbw.fabbwled.lands.book.SectionNode;
+import ch.bbw.fabbwled.lands.character.PlayerDto;
+import ch.bbw.fabbwled.lands.service.PlayerSession;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+
+@Component
+@RequiredArgsConstructor
+public class Section10 implements SectionHandler {
+
+    @Override
+    public SectionId getId() {
+        return new SectionId(1, 10);
+    }
+
+
+    @Override
+    public SectionNode getBody(PlayerDto current) {
+        return SectionNode.root().text("""
+                        If you have just arrived in Yellowport, tick the first empty box above (use a pencil).
+                        Yellowport is the second largest city in Sokara.
+                        It is mainly a trading town, and is known for its exotic goods from distant Ankon-Konu, way to the south.
+                        The Stinking River brings rich deposits of sulphur from the Lake of the Sea Dragon down to the town, where it is extracted and stored in the large waterfront warehouses run by the merchants’ guild.
+                        From here, the mineral is exported all over Harkuna.
+                        Unfortunately, all that sulphur has its drawbacks.
+                        The stink is abominable, and much of the city has a yellowish hue.
+                        The river is so full of sulphur that it is virtually useless as a source of food or of drinking water.
+                        However, the demand for sulphur, especially from the sorcerous guilds, is great. Politically, much has changed in the past few years.
+                        The old and corrupt king of Sokara, Corin VII, has been deposed and executed in a military coup. General Grieve Marlock and the army now control Sokara.
+                        The old Council of Yellowport has been ‘indefinitely dissolved’ and a provost marshal, Marloes Marlock, the general’s brother, appointed as military governor of the town.
+
+                        Owning a house gives you a place to rest, and to store equipment.
+                        If you buy one, tick the box by the town house option and cross off 200 Shards from your Adventure Sheet.
+                        To leave Yellowport by sea, buy or sell ships and cargo, go to the harbourmaster.
+                              """)
```

Comment:
you need to be careful with the closing quotes for multiline strings, they should be on the _same_ column as the text.